front/src/common/Map/Search/MapSearchStation.tsx

```diff
@@ -23,9 +26,14 @@ type MapSearchStationProps = {
 const MapSearchStation = ({ updateExtViewport, closeMapSearchPopUp }: MapSearchStationProps) => {
   const map = useSelector(getMap);
   const [searchState, setSearch] = useState<string>('');
+  const [chCodeFilter, setChCodeFilter] = useState<string>('');
+  const [bvOnly, setBvOnly] = useState<boolean>(false);
```

Comment:
You don't need to set the type. And I find it clearer when we use toggle for a boolean state. What do you think ?
```suggestion
  const [bvOnly, toggleBvOnly] = useState(false);
```